cypress/integration/createBoard.spec.js

```diff
@@ -0,0 +1,259 @@
+/// <reference types ="Cypress" />
+import board from "../fixtures/addBoard.json"
+import org from "../fixtures/addOrg.json"
+import login from "../fixtures/login.json"
+import data from "../fixtures/data.json"
+import faker from "faker"
+import url from "../fixtures/url.json"
+import sideBar from "../fixtures/sideBar.json"
+//nisam usela da koristim import za ove fajlove jer dobijam error u cypressu
+const jpg = "../fixtures/media/valid/jpg.jpg"
+const gif = '../fixtures/media/valid/gif.gif';
+const jpeg = '../fixtures/media/valid/jpeg.jpeg';
+const png = '../fixtures/media/valid/png.png';
+const ai = '../fixtures/media/invalid/ai.ai';
+const bmp = '../fixtures/media/invalid/bmp.bmp';
+const docx = '../fixtures/media/invalid/docx.docx';
+const eps = '../fixtures/media/invalid/eps.eps';
+const pdf = '../fixtures/media/invalid/pdf.pdf';
+const psd = '../fixtures/media/invalid/psd.psd';
+const svg = '../fixtures/media/invalid/svg.svg';
+const tiff = '../fixtures/media/invalid/tiff.tiff';
+const zip = '../fixtures/media/invalid/zip.zip';
```

Comment:
There is absolutely no need to import all these files. Testing file upload is way too far ahead of time at this moment.

Reply:
It would be a shame to delete it now :(